packages/teams-js/src/private/privateAPIs.ts

```diff
@@ -5,13 +5,12 @@ import { registerHandler } from '../internal/handlers';
 import { ensureInitialized } from '../internal/internalAPIs';
 import { getGenericOnCompleteHandler } from '../internal/utils';
 import { UserSettingTypes } from './interfaces';
-import { menus } from './menus';
 
 /**
  * @internal
  */
 export function initializePrivateApis(): void {
-  menus.initialize();
+  //Keeping this API for any future usage, wherein privateAPIs need to be initialized
```

Comment:
Do we have this logged as a work item?

Reply:
Not yet. I plan to log a few WIs today as you can see there are new export functions added as well. I will post details in our project channel when the PR is ready for review. Thanks for checking it.